### PR TITLE
Feat: Abstract encryption/decryption and ipfs storage/retrieval

### PR DESCRIPTION
**Pull request by Adarsh-Kumar28**
# What

Abstract all the multiple steps for encryption, storing the encrypted metadata & decryption into 2 functions, one for encryption & storing on IPFS & the other for retrieving from IPFS & using it to decrypt the encrypted content.

# Why

Since most devs > 60% store the encryptedString/File & metadata on IPFS. Also, both encryption & decryption requires multiple steps. Abstracting all these steps which improve the developed experience using the SDK.

# How

- Abstracting away the multiple steps in a single function
- Users pass their Infura Project Id & API Secret Key to the `encryptAndUploadMetadataToIpfs` function

# Additional Comments

Used error handling instead of returning `undefined`.

# Tests

I've tested this branch on both a Node.js & React (TS) app. I tested it the following way:
`yarn build:packages && cd packages/encryption && npm link && cd ../..` then `npm link @lit-protocol/encryption` in my test apps.

**Working repo: https://github.com/Adarsh-Kumar28/Encrypt-File-Encryption**

### CODE REVIEW

File: packages/encryption/src/lib/encryption.ts [hunk 1]
```diff
@@ -128,18 +128,31 @@ export const encryptAndUploadMetadataToIpfs = async ({
...
 
-  if (!string && !file) return;
+  if (string === undefined && file === undefined) return throwError({
+    message: `Either string or file must be provided`,
+    error: LIT_ERROR.INVALID_PARAM_TYPE,
+  });
+
+  //   @ts-ignore
+  if (!process.env.INFURA_ID || !process.env.INFURA_SECRET_KEY) {
+    throw new Error('Please provide your INFURA_ID and INFURA_SECRET_KEY as env variables');
```
Comment (Ansonhkg) [inline packages/encryption/src/lib/encryption.ts]: Please use the throwError function throughout the SDK.

**Comment by Ansonhkg** [comment]
Please add notable change here:
https://github.com/LIT-Protocol/js-sdk/blob/master/CHANGELOG.md

**Comment by Ansonhkg** [comment]
Published version 2.1.83 as per @Adarsh-Kumar28 request

# Test

```
yarn add @lit-protocol/lit-node-client@2.1.83
```

**Comment by Adarsh-Kumar28** [comment]
> Published version 2.1.83 as per @Adarsh-Kumar28 request
> 
> # Test
> ```
> yarn add @lit-protocol/lit-node-client@2.1.83
> ```

Test on both React & Node.js app! @Ansonhkg

**Comment by glitch003** [comment]
How is this looking @Adarsh-Kumar28 and @Ansonhkg  ? Can we merge it soon?  I don't wanna let it get stale :)

**Comment by Adarsh-Kumar28** [comment]
@glitch003 Merged!

**Pull request merged**